packages/dd-trace/test/ci-visibility/exporters/ci-visibility-exporter.spec.js

```diff
@@ -676,9 +676,7 @@ describe('CI Visibility Exporter', () => {
         ciVisibilityExporter._resolveCanUseCiVisProtocol(false)
         ciVisibilityExporter._libraryConfig = { isEarlyFlakeDetectionEnabled: true }
         ciVisibilityExporter.getKnownTests({}, (err) => {
-          expect(err.message).to.include(
-            'Known tests can not be requested because CI Visibility protocol can not be used'
```

Comment:
no need to do this: if CI Vis protocol can't be used, the library settings can't be fetched, so we will not even reach this situation